backend/common/src/main/java/ai/verta/modeldb/common/db/migration/Migration.java

```diff
@@ -3,7 +3,7 @@
 import lombok.Value;
 
 @Value
-class Migration implements Comparable<Migration> {
+public class Migration implements Comparable<Migration> {
```

Comment:
not too thrilled with making this stuff public, but I see why you needed it. Once things are stable, we should remove the single-migration tests from the server module, and just rely on the standard integration tests to validate. Then, we can tighten this back up again.